schema.sql

```diff
@@ -0,0 +1,148 @@
+CREATE DATABASE sdc_products;
+
+USE sdc_products;
+
+DROP TABLE IF EXISTS products;
+
+CREATE TABLE products (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  item_name TEXT,
+  slogan TEXT,
+  item_description TEXT,
+  category VARCHAR(100),
+  default_price INTEGER
+);
+
+DROP TABLE IF EXISTS features;
+
+CREATE TABLE features (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT NOT NULL,
+  feature TEXT,
+  item_value TEXT,
+  product_id INTEGER,
+    FOREIGN KEY (product_id)
+      REFERENCES products (id)
+);
+
+DROP TABLE IF EXISTS styles;
+
+CREATE TABLE styles (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  product_id INTEGER,
+    FOREIGN KEY (product_id)
+      REFERENCES products (id),
+  style_name TEXT,
+  sale_price VARCHAR(6),
+  original_price VARCHAR(6),
+  photos INTEGER,
+  default_style TINYINT(1) DEFAULT 0
+);
+
+DROP TABLE IF EXISTS photos;
+
+CREATE TABLE photos (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  thumbnail_url TEXT,
+  url TEXT,
+  style_id INTEGER,
+    FOREIGN KEY (style_id)
+      REFERENCES styles (id)
+);
+
+
+DROP TABLE IF EXISTS skus;
+
+CREATE TABLE skus (
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
+  quantity INTEGER,
+  size VARCHAR(1),
```

Comment:
Is VARCHAR(1) large enough? I believe the data set contained sizes like XS, XL, XXL.

Reply:
Confirmed, thank you!  This has been amended.